rules/klayout/drc/testing/run_sc_regression.py

```diff
@@ -76,15 +77,15 @@ def call_simulator(arg):
     :param path: The path to the GDS file you want to simulate
     :param thrCount: number of threads to use
     """
-    os.system(arg)
+    subprocess.Popen(["klayout"] + arg.split(" ")).wait()
 
 if __name__ == "__main__":
 
     t0 = time.time()
     args = docopt(__doc__)
 
     if os.path.exists("sc"):
-        os.system("rm -rf sc")
+        subprocess.run(["rm", "-rf", "sc"])
```

Comment:
This should use `shutils` and not call out to `rm`.

Reply:
@mithro Totally agree. We will fix that.
@mithro As we have discussed earlier, there will be a major change to how we run the test cases, after the device renaming is pushed in, to make each unit test run on its own.

@mohanad0mohamed Please make sure to use shutils instead of calling out rm directly.